packages/core/src/components/HorizontalScrollGrid/HorizontalScrollGrid.test.tsx

```diff
@@ -23,9 +23,10 @@ import { Grid } from '@material-ui/core';
 describe('<HorizontalScrollGrid />', () => {
   beforeEach(() => {
     jest.spyOn(window.performance, 'now').mockReturnValue(5);
-    jest
-      .spyOn(window, 'requestAnimationFrame')
-      .mockImplementation(cb => cb(20));
+    jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => {
+      cb(20);
+      return 1;
```

Comment:
typescript really wanted a return value here - am actually not sure why that's needed

Reply:
I think it's the number that you can use to cancel the request for the animation frame.